db/migrate/20170629203747_add_new_identifier_tables.rb

```diff
@@ -0,0 +1,20 @@
+class AddNewIdentifierTables < ActiveRecord::Migration[5.1]
+  def change
+    create_table :rosters do |t|
+      t.string :identifier_name, null: false
```

Comment:
Almost certainly not, the identifier_name is just a human readable description of what the entry identifiers represent. e.g. 'your student email'